keras/applications/mobilenet.py

```diff
@@ -0,0 +1,275 @@
+from keras.layers.pooling import GlobalAveragePooling2D, GlobalMaxPooling2D
+from keras.layers import Input
+from keras.layers.normalization import BatchNormalization
+from keras.utils.data_utils import get_file
+from keras.engine.topology import get_source_inputs
+from keras.applications.imagenet_utils import _obtain_input_shape
+import keras.backend as K
+
+
+BASE_WEIGHT_PATH = 'https://github.com/titu1994/MobileNetworks/releases/download/v1.0/'
+
+
+def MobileNet(input_shape=None, alpha=1.0, depth_multiplier=1,
+              dropout=1e-3, include_top=True, weights='imagenet',
+              input_tensor=None, pooling=None, classes=1000):
+    ''' Instantiate the MobileNet architecture.
+        Note that only TensorFlow is supported for now,
+        therefore it only works with the data format
+        `image_data_format='channels_last'` in your Keras config
+        at ~/.keras/keras.json.
+        # Arguments
+            input_shape: optional shape tuple, only to be specified
+                if `include_top` is False (otherwise the input shape
+                has to be `(224, 224, 3)` (with `channels_last` data format)
+                or (3, 224, 224) (with `channels_first` data format).
+                It should have exactly 3 inputs channels,
+                and width and height should be no smaller than 32.
+                E.g. `(200, 200, 3)` would be one valid value.
+            alpha: width multiplier of the MobileNet.
+            depth_multiplier: depth multiplier for depthwise convolution
+                (also called the resolution multiplier)
+            dropout: dropout rate
+            include_top: whether to include the fully-connected
+                layer at the top of the network.
+            weights: `None` (random initialization) or
+                `imagenet` (ImageNet weights)
+            input_tensor: optional Keras tensor (i.e. output of `layers.Input()`)
+                to use as image input for the model.
+            pooling: Optional pooling mode for feature extraction
+                when `include_top` is `False`.
+                - `None` means that the output of the model will be
+                    the 4D tensor output of the
+                    last convolutional layer.
+                - `avg` means that global average pooling
+                    will be applied to the output of the
+                    last convolutional layer, and thus
+                    the output of the model will be a 2D tensor.
+                - `max` means that global max pooling will
+                    be applied.
+            classes: optional number of classes to classify images
+                into, only to be specified if `include_top` is True, and
+                if no `weights` argument is specified.
+        # Returns
+            A Keras model instance.
+        '''
+
+    if K.backend() != 'tensorflow':
+        raise AttributeError('Only Tensorflow backend is currently supported, '
```

Comment:
Yes it should. I consider `tensorflow` an attribute value for `backend`, therefore the Attribute error. 
Fixed.